infra/presubmit.py

```diff
@@ -346,8 +360,12 @@ def main():
     success = check_license(changed_files)
     return bool_to_returncode(success)
 
+  if args.command == 'test':
+    return run_tests()
+
   # Otherwise, do all of them.
```

Comment:
nit: can you update this comment to say "do all the checks (but no tests)"?

Reply:
Added.